build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/CheckApiCompatibility.cs

```diff
@@ -165,7 +165,7 @@ void ValidateApiCompat (string contractPath, bool validateAgainstReference)
 				using (var genApiProcess = new Process ()) {
 
 					if (Environment.Version.Major >= 5) {
-						var apiCompat = new FileInfo (Path.Combine (ApiCompatPath, "..", "netcoreapp2.1", "Microsoft.DotNet.ApiCompat.dll"));
+						var apiCompat = new FileInfo (Path.Combine (ApiCompatPath, "..", "netcoreapp3.1", "Microsoft.DotNet.ApiCompat.dll"));
```

Comment:
I got xamarin-android to build on M1 the other day (first time!).

This is one of the apps that fails, maybe the new version will work?